cpp/src/slice2java/Gen.cpp

```diff
@@ -1555,18 +1555,24 @@ Slice::JavaVisitor::writeHiddenProxyDocComment(Output& out, const OperationPtr&
     // the _iceI_ methods are all async
 
     out << nl << "/**";
-    out << nl << " * @hidden";
+
+    out << nl << " * Invokes the " << p->name()
```

Comment:
Did you try to generate instead `@SuppressWarnings("doclint:missing")` ?

It's Java 18, but maybe we could run Javadoc 18+ to generate the API reference?

https://bugs.openjdk.org/browse/JDK-8275351

Reply:
I didn't try it. While it might work we support Java 17 so it won't work for our users.